.pre-commit-config.yaml

```diff
@@ -34,6 +34,7 @@ repos:
           - numpy==2.1  # until https://github.com/numpy/numpy/issues/28034 is resolved
           - typing_extensions
           - universal-pathlib
+          - obstore==0.3.0-beta.8
```

Comment:
TODO: update to latest obstore release when we merge

Reply:
Note: obstore is currently on version 0.4.0-beta.4. I'm ready to publish 0.4.0 but wanted to ensure it works with zarr-python before publishing it.

Reply:
Done